packages/emails/tsconfig.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tsconfig.json",
+}
```

```suggestion
{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
    "declaration": true
  }
}

```

Esse pacote precisa declarar tipos...